lib/yabeda/prometheus/version.rb

```diff
@@ -2,6 +2,6 @@
 
 module Yabeda
   module Prometheus
-    VERSION = "0.8.0"
+    VERSION = "0.9.0"
```

Comment:
Please don't bump version numbers. I will do it myself on release

Reply:
Awesome I’ll remove that change from this MR